Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -82,6 +82,24 @@ public UIPointerBehavior pointerBehavior
             set => m_PointerBehavior = value;
         }
 
+        /// <summary>
+        /// Where to position the pointer when the cursor is locked.
+        /// </summary>
+        /// <remarks>
+        /// By default, the pointer is positioned at -1, -1 in screen space when the cursor is locked. This has implications
```

Comment:
I know I asked this before but just to make 100% extra super sure... `StandaloneInputModule` does not do the raycast-from-center thing? If it does, default is probably better the other way round.

Reply:
Double checked. StandaloneInputModule also sets the raycast origin to -1, -1.

Reply:
Cool, thank you :)